### PR TITLE
ft(upload-imges): Configure cloudinary

### PR DESCRIPTION
**Pull request by abraham-kwizera**
## What does this PR do?

> Update Profile Picture & Configure Cloudinary

## Description of Task to be completed?

- [x] Update Users model with profilePicture field
- [x] update seeders
- [x] install cloudinary package
- [x] Update the updateProfile route with profilePicture field for holding images
- [x] Implements setups in update profile endpoint, of the upload presets to where images will be located
- [x] Retouch on sequelize instance
- [x] Remove conflicts referring to base branch, that has resulted in sequelize instance error

### How should this be manually tested?

step1: clone git repo if not yet done
step2: `cd Avengers-Phantom-BE`
step3: `npm i` or `npm install` to install dependencies
step4: `npm run start:dev` to start the server

#### API

{{URL}}api/v1/users/updateProfile/:uuid

[POSTMAN](https://www.postman.com/) to test all the above APIs

### CODE REVIEW

File: migrations/20220429162252-create-user.js [hunk 1]
```diff
@@ -71,6 +71,20 @@ module.exports = {
       roleName: {
         type: DataTypes.STRING,
       },
+      roleId: {
+        type: DataTypes.INTEGER,
+      },
+      roleName: {
+        type: DataTypes.STRING,
+      },
+      isAssigned: {
```
Comment (capitale250) [inline migrations/20220429162252-create-user.js:80]: we have two isassigned columns and also rolename and others

**Review by capitale250 — CHANGES_REQUESTED** [review]
and also we have two files in migration wich serves the same role

**Review by Gedeonir — CHANGES_REQUESTED** [review]
- Changing the response messages will cause the tests written before to fail. try to not change it especialy on roles,buses,and authentication files.

- we are no longer using config.json because it exposes database info we are using config.js file and save info in .env file and failing to do so it will make our CIs to fail.
- again to folder structure has changed we no longer have tests stored in test folder instead every test is in its corresponding folder(ex: test file for roles is in Role folder)
- do well your rebase just pull develop to get latest content. some files still has content of from the last month whereas they are changed now like phantom.yml in github/workflows
-

**Review by un-anelka — COMMENTED** [review]
Just do rebase on the current changes on the develop to have updated project structure and try to use config.js instead of config.json file and finally do squash your commits.

File: migrations/20220429162252-create-user.js (hunk 1 above)
Comment (abraham-kwizera) [reply migrations/20220429162252-create-user.js:80]: Well noted. Solved.

**Pull request closed without merging**